modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/getContentType.js

```diff
@@ -0,0 +1,26 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+// BEGIN
+// Fetch virtual content type.
+var result = schemaLib.getSchema({
+    name: 'myapp:mytype',
+    type: 'CONTENT_TYPE'
+});
+
+log.info('Fetched content type: ' + result.name);
+
+// END
+
+
+assert.assertJsonEquals({
+    name: "myapp:mytype",
+    displayName: "My type display name",
+    description: "My type description",
+    modifiedTime: "2010-01-01T10:00:00Z",
+    resource: "<content-type><some-data></some-data></content-type>",
+    type: "CONTENT_TYPE"
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    type: 'CONTENT_TYPE'
```